sample-app/app/assets/stylesheets/custom.scss

```
@@ -0,0 +1,112 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+/* mixins, variables, etc. */
+$gray-medium-light: #eaeaea;
+$light-black:#333333;
+/* universal */
+body {
+  padding-top: 60px;
+}
```

sau mỗi block css thì e cách dòng ra nhé. c vd:
```suggestion
}

section {
  overflow: auto;
}

textarea {
  resize: vertical;
}
```